systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -595,6 +595,11 @@ public static String changeOrgAndTagInImageMap(String imageMap) {
         return sb.toString();
     }
 
+    public static void waitUntilSecretsChange() throws InterruptedException {
```

Comment:
How stable will this be when it really just sleeps for 4 seconds? Also, why just not do `Thread.sleep(4000)` directly in the test? I that most of the other methods with similar names are actually checking the secret resources to change so this can lead to confusion.

Reply:
I experimented with the 1 to 10 seconds. I used 3s and it fails because it was really short time. Agree, i will change it.